shared/js/ui/models/submit-breakage-form.es6.js

```diff
@@ -25,13 +25,15 @@ module.exports = function (category) {
     const blockedTrackers = collectAllUrls(aggregationStats.blockAction.list)
     const surrogates = collectAllUrls(aggregationStats.redirectAction.list)
     const urlParametersRemoved = this.tab.urlParametersRemoved ? 'true' : 'false'
+    const ctlYouTubeAdded = this.tab.ctlYouTubeAdded ? 'true' : 'false'
```

Comment:
Nit: Cool use Boolean again here?
```suggestion
    const ctlYouTubeAdded = Boolean(this.tab.ctlYouTubeAdded)
```